rgbd/src/get_3d_point_from_image_roi_node.cpp

```diff
@@ -130,7 +132,7 @@ int main(int argc, char **argv)
     ros::Time last_image_stamp;
 
     ros::Rate r(max_fps);
-    while (ros::ok())
+    while (ros::ok() && ros::master::check())
```

Comment:
If `ros::master::check()` does not return true, something's wrong and I'd like to see an error message (and `exit(1)`)